org.bardsoftware.eclipsito/src/org/bardsoftware/test/eclipsito/TestPluginFolderResolver.java

```diff
@@ -0,0 +1,62 @@
+package org.bardsoftware.test.eclipsito;
```

Comment:
Please add copyright notice :)

Reply:
I saw that all classes in eclipsito repository haven't copyright notice
but of course, I'll add it)